README.md

```diff
@@ -47,25 +45,27 @@ const secrets = loadSecrets();
 
 ## Background and Motivation
 
-As creators of the [Doppler Universal Secrets Platform](https://www.doppler.com/) who provide secrets sync integrations for [Vercel](https://vercel.com/integrations/doppler) and [Serverless](https://docs.doppler.com/docs/enclave-installation-serverless), we've helped customers individually to work around this limitation.
+Exceeding AWS Lambda's 4KB environment variable limit is a common problem that also impacts platforms such as [Vercel](https://vercel.com/support/articles/how-do-i-workaround-vercel-s-4-kb-environment-variables-limit) and the [Serverless framework](https://www.serverless.com/framework/docs/providers/aws/guide/variables) who deploy on top of AWS Lambda.
```

Comment:
copy style, optional: "who deploy on top of AWS Lambda" -> "which deploy on top of AWS Lambda". I'll leave it to you though